projects/demo/src/modules/components/input-files/examples/3/index.html

```diff
@@ -0,0 +1,44 @@
+<tui-files>
+    <ng-container *ngFor="let file of files">
+        <tui-file *tuiFile state="normal" [file]="file"></tui-file>
+    </ng-container>
+
+    <ng-container *ngFor="let file of rejectedFiles">
+        <tui-file *tuiFile state="error" [file]="file"></tui-file>
+    </ng-container>
+
+    <ng-container *ngIf="loadingFile">
+        <tui-file
+            *tuiFile
+            state="loading"
```

Comment:
This blinking loader is potentially new flaky screenshot test.

Let's make this example more stable using `TUI_IS_CYPRESS`